### PR TITLE
Hotfix/#357 spacing

### PR DESCRIPTION
**Pull request by NicolaSaunders**
Suggested fixes for https://github.com/w3c/w3c-website/issues/357

Includes some updates and corrections to the design system documentation to reflect previous changes

### CODE REVIEW

**Comment by deniak** [comment]
That PR addresses the issue regarding spacing. However, it also includes quite a few markup changes on components we are already using, so I'd like to ask @jean-gui to also review that PR as it might require some work on our side to adapt the new markup

File: design-system-templates/components/activity.html.twig [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="component component--activity u-full-width">
     <div class="l-center">
         <div class="component component--columns component--columns--images">
-            <div class="component--columns__intro">
+            <div>
```
Comment (vivienlacourba) [inline design-system-templates/components/activity.html.twig:4]: Is that now class-less div necessary?

File: design-system-templates/example-pages/landing.html.twig [hunk 2]
```diff
@@ -14,7 +14,7 @@
 	{% include 'components/hero-landing.html.twig' %}
 
 	<div class="component component--columns component--columns--images">
-		<div class="component--columns__intro">
+		<div>
```
Comment (vivienlacourba) [inline design-system-templates/example-pages/landing.html.twig:17]: Same comment: Is that now class-less div necessary?

File: templates/components/columns_with_icons.html.twig [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <div class="component component--columns component--columns--icons">
 	{% if title is defined and title or summary is defined and summary %}
-	<div class="component--columns__intro">
+	<div>
```
Comment (vivienlacourba) [inline templates/components/columns_with_icons.html.twig:3]: Same comment: Is that now class-less div necessary?

File: templates/components/columns_with_images.html.twig [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <div class="component component--columns component--columns--images">
 	{% if title is defined and title or summary is defined and summary %}
-		<div class="component--columns__intro">
+		<div>
```
Comment (vivienlacourba) [inline templates/components/columns_with_images.html.twig:3]: Same comment: Is that now class-less div necessary?

File: templates/components/recent-activities.html.twig [hunk 5]
```diff
@@ -2,7 +2,7 @@
     <div class="l-center">
         <div class="component component--columns component--columns--images">
             {% if title is not empty or introduction is not empty %}
-            <div class="component--columns__intro">
+            <div>
```
Comment (vivienlacourba) [inline templates/components/recent-activities.html.twig:5]: Same comment: Is that now class-less div necessary?

File: assets-src/styles/sass/30-base/_tables.scss [hunk 6]
```diff
@@ -7,6 +7,8 @@
 table {
     border: 1px solid $border-color;
     border-collapse: collapse;
+	margin-bottom: 1rem;
+	margin-top: 1rem;
     width: 100%;
```
Comment (vivienlacourba) [inline assets-src/styles/sass/30-base/_tables.scss:12]: Code indentation needs fixing.

File: assets-src/styles/sass/30-base/_tables.scss [hunk 7]
```diff
@@ -52,6 +54,8 @@ https://adrianroselli.com/2020/11/under-engineered-responsive-tables.html
 @media screen {
     .table-wrap {
         border: 1px solid $border-color;
+		margin-bottom: 1rem;
+		margin-top: 1rem;
         max-width: 100% !important;
```
Comment (vivienlacourba) [inline assets-src/styles/sass/30-base/_tables.scss:59]: Same comment: code indentation needs fixing.

File: assets-src/styles/sass/30-base/_tables.scss [hunk 8]
```diff
@@ -60,6 +64,8 @@ https://adrianroselli.com/2020/11/under-engineered-responsive-tables.html
 
         table {
             border: 0;
+			margin-bottom: 0;
+			margin-top: 0;
```
Comment (vivienlacourba) [inline assets-src/styles/sass/30-base/_tables.scss]: Same comment: code indentation needs fixing.

File: design-system-templates/example-pages/landing.html.twig (hunk 2 above)
Comment (NicolaSaunders) [reply design-system-templates/example-pages/landing.html.twig]: Removed in latest commits

File: assets-src/styles/sass/30-base/_tables.scss (hunk 6 above)
Comment (NicolaSaunders) [reply assets-src/styles/sass/30-base/_tables.scss]: Please re-check after pulling latest commits. I'm not sure if my default IDE indent spacing has changed between spaces and tabs.

File: assets-src/styles/sass/30-base/_tables.scss (hunk 7 above)
Comment (NicolaSaunders) [reply assets-src/styles/sass/30-base/_tables.scss]: Please re-check after pulling latest commits. I'm not sure if my default IDE indent spacing has changed between spaces and tabs.

File: assets-src/styles/sass/30-base/_tables.scss (hunk 8 above)
Comment (NicolaSaunders) [reply assets-src/styles/sass/30-base/_tables.scss]: Please re-check after pulling latest commits. I'm not sure if my default IDE indent spacing has changed between spaces and tabs.

**Comment by deniak** [comment]
@NicolaSaunders The PR looks good. Can you please resolve the conflicts listed above and I'll take care of merging it and updating the frontend and the main app.

**Comment by NicolaSaunders** [comment]
@deniak That should be all the conflicts resolved now

**Pull request merged**